frontend/src/metabase/components/MetadataInfo/TableInfoIcon/TableInfoIcon.unit.spec.tsx

```diff
@@ -0,0 +1,42 @@
+import { createMockEntitiesState } from "__support__/store";
+import { fireEvent, renderWithProviders, screen } from "__support__/ui";
+import type { Table } from "metabase-types/api";
+import { createMockTable } from "metabase-types/api/mocks";
+import { createMockState } from "metabase-types/store/mocks";
+
+import { TableInfoIcon } from "./TableInfoIcon";
+
+type SetupOpts = {
+  table: Table;
+};
+
+function setup({ table }: SetupOpts) {
+  const state = createMockState({
+    entities: createMockEntitiesState({
+      tables: table ? [table] : [],
```

Comment:
`table` is required in `SetupOpts`

```suggestion
      tables: [table],
```